imports/client/components/DocumentDisplay.tsx

```diff
@@ -19,10 +19,10 @@ const StyledDeepLink = styled(DeepLink)`
 `;
 
 const StyledIframe = styled.iframe`
-  /* Workaround for unusual sizing behavior of iframes in iOS Safari */
-  /* Width and height need to be specified in px then adjusted by min and max */
-  width: 0px;
-  height: 0px;
+  /* Workaround for unusual sizing behavior of iframes in iOS Safari:
+   * Width and height need to be specified in px then adjusted by min and max */
+  width: 0;
```

Comment:
Given the comment about what we're trying to work around, it's not totally clear to me that switching from `px` to unitless is actually safe here -- can we verify that this still does what we intended?

Reply:
Based on [the commit that introduced this formatting](https://github.com/deathandmayhem/jolly-roger/commit/9a1a35f034f90b02aa2661a2dcac3c84ec4dfc00) and [this StackOverflow response](https://stackoverflow.com/a/23083463), I believe the relevant thing here is moving from `height: 100%; width: 100%` to using `{min,max}-{width,height}`, and the comment should probably say that "width and height need to be specified as an absolute value instead of a percentage."

However, I did borrow @ccpost's iPad to see if I could reproduce this issue, and I was unable to. I tried the configuration on this branch, the configuration on master, and I also tried reverting the change in 9a1a35f034f90b02aa2661a2dcac3c84ec4dfc00. The rendering and scrolling behavior in all three cases seemed to be the same. I'm not sure if I'm misunderstanding what the original issue was or if Apple has since changed WebKit's behavior.

I think that means we could probably revert that change - it would have the minor advantage of simplifying the styling some - but I'm not going to do that for now without a better understanding off the issue. However, I do think it's safe to leave the dimensions unitless, and I'll update the comment to be more clear.